tasks/src/main/java/ru/sberbank/school/task07/Counter.java

```diff
@@ -0,0 +1,20 @@
+package ru.sberbank.school.task07;
+
+
+import java.io.FileNotFoundException;
+
+/**
+ * Считатель различных слов в файле.
+ * <p>

```

Comment:
&lt;p&gt; tag should be preceded with an empty line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1954513551>*